api/src/main/resources/underlays/aou_synthetic.yaml

```diff
@@ -424,116 +424,117 @@ uiConfiguration: >-
     },
     "criteriaConfigs": [{
       "type":"concept",
+      "id": "tanagra-conditions",
       "title":"Conditions",
       "defaultName":"Contains Conditions Codes",
-      "plugin": {
-        "columns": [
-          {"key":"concept_name","width":"100%","title":"Concept Name"},
-          {"key":"concept_id","width":120,"title":"Concept ID"},
-          {"key":"standard_concept","width":180,"title":"SourceStandard"},
-          {"key":"vocabulary_id","width":120,"title":"Vocab"},
-          {"key":"concept_code","width":120,"title":"Code"},
-          {"key":"person_count","width":120,"title":"Roll-up Count"}
-        ],
-        "hierarchyColumns": [
-          {"key": "concept_name", "width": "100%", "title": "Condition"},
-          {"key": "concept_id", "width": 120, "title": "Concept ID"},
-          {"key":"person_count","width":120,"title":"Roll-up Count"}
-        ],
-        "occurrence": "condition_occurrence",
-        "classification": "condition"
-      }
+      "columns": [
+        {"key":"concept_name","width":"100%","title":"Concept Name"},
+        {"key":"concept_id","width":120,"title":"Concept ID"},
+        {"key":"standard_concept","width":180,"title":"SourceStandard"},
+        {"key":"vocabulary_id","width":120,"title":"Vocab"},
+        {"key":"concept_code","width":120,"title":"Code"},
+        {"key":"person_count","width":120,"title":"Roll-up Count"}
+      ],
+      "hierarchyColumns": [
+        {"key": "concept_name", "width": "100%", "title": "Condition"},
+        {"key": "concept_id", "width": 120, "title": "Concept ID"},
+        {"key":"person_count","width":120,"title":"Roll-up Count"}
+      ],
+      "occurrence": "condition_occurrence",
+      "classification": "condition"
```

Comment:
Do these both correspond to the entity names?

Reply:
I've generally captured the underlying entities separately from the the concepts that use them. For occurrences, that's probably always a direct mapping that can be merged into the entity definitions. For classifications, they currently all map directly to entities but I think that won't always be the case. In particular I'm thinking of how the source/standard trees work whether they are two separate ways to classify entities like conditions.